html/browsers/windows/clear-window-name.https.html

```diff
@@ -41,66 +41,66 @@
   });
 }
 
-async_test(async t => {
+promise_test(async t => {
   const id = token();
 
   window.open(`resources/window-name.sub.html?report=${id}|close`, id);
   await pollResultAndCheck(t, id, id);;
```

Comment:
The `pollResultAndCheck` helper is still quite broken. It shouldn't call `t.done()` at the very least, and has `t.step_func(async () => ...)` which is broken in a similar way to the tests you're fixing.

Reply:
Whoops! Done.